#
#
# patch "libpurple/protocols/msn/slplink.c"
#  from [a96a53635871463de0177e5a99fe4ebdc2c7d577]
#    to [87ada76abf90c44e615679efc5f8128bb941bba1]
#
============================================================
--- libpurple/protocols/msn/slplink.c	a96a53635871463de0177e5a99fe4ebdc2c7d577
+++ libpurple/protocols/msn/slplink.c	87ada76abf90c44e615679efc5f8128bb941bba1
@@ -383,6 +383,7 @@ msn_slplink_release_slpmsg(MsnSlpLink *s
 msn_slplink_release_slpmsg(MsnSlpLink *slplink, MsnSlpMessage *slpmsg)
 {
 	MsnMessage *msg;
+	const char *passport;
 
 	slpmsg->msg = msg = msn_message_new_msnslp();
 
@@ -421,7 +422,8 @@ msn_slplink_release_slpmsg(MsnSlpLink *s
 
 	msg->msnslp_header.total_size = slpmsg->size;
 
-	msn_message_set_attr(msg, "P2P-Dest", slplink->remote_user);
+	passport = purple_normalize(slplink->session->account, slplink->remote_user);
+	msn_message_set_attr(msg, "P2P-Dest", passport);
 
 	msg->ack_cb = msg_ack;
 	msg->nak_cb = msg_nak;

